graphql-samples/mutations/collaboration/create-aspect-on-callout

```diff
@@ -0,0 +1,19 @@
+mutation createAspectOnCallout($data: CreateAspectOnCalloutInput!){
```

Comment:
This should be under `graphql-samples/mutations/create/create-aspect-on-callout`
Also, remove the redundant `create-aspect-on-context` there.